.vscode/settings.json

```diff
@@ -1,3 +1,14 @@
 {
-    "go.buildFlags": ["-tags=unit,integration"],
+    "go.buildFlags": [
+        "-tags=unit,integration"
+    ],
+    "files.exclude": {
+        "**/.git": true,
+        "**/.svn": true,
+        "**/.hg": true,
```

Comment:
Why we need to exclude svg and mercurial file formats?

Reply:
Sure, *svn, we don't want them committed also don't want them to be indexed or considered in any way by vscode.

Reply:
So you will list all the file formats? :D

Reply:
Only those to be skipped by search and the IDE, this list is probably from some suggestion.